index.html

```diff
@@ -16,6 +16,11 @@
   <body>
     <header>
       <a href="#">My Logo</a>
+      <ul>
+        <li>Portfolio</li>
+        <li>About</li>
+        <li class="last">Contact</li>
+      </ul>
       <nav>
```

Comment:
Nice job putting the nav inside the header! Please wrap your `ul` with the `nav` element that you currently have. This will help with the accessibility & semantics of your project. 😄 You should have something like:
<_nav_>
<_ul_>
your li elements
<_/ul_>
your hamburger button
<_/nav_>